resources/js/processes/screen-templates/components/MyTemplatesListing.vue

```diff
@@ -213,6 +213,27 @@ export default {
           this.loading = false;
         });
     },
+    onTemplateNavigate(action, data, index) {
```

Comment:
hi @estebangallego! To address the duplicate code issues identified by Sonarqube, it will be necessary to move this code to the mixin located at `resources/js/processes/screen-templates/mixins/navigationMixin.js`. Please check that file

Reply:
Thank you @eiresendez! I've resolved the Sonarqube issues.